charts/spire/templates/server-statefulset.yaml

```diff
@@ -75,7 +75,28 @@ spec:
             periodSeconds: 5
           resources:
             {{- toYaml .Values.server.resources | nindent 12 }}
-        - name: {{ .Chart.Name }}-workload-registrar
+        {{- if .Values.controllerManager.enabled }}
```

Comment:
Can we isolate this into a separate `Deployment`/file? I assume it doesn't need persistence from code below. In case it needs persistence, please make it a `StatefulSet`.

Probably also include the wait-for-it initContainer to ensure spire server is up and running when launching the controller manager.

Reply:
I've created another deployment file for spire-controller-manager